sailfishusermanager.cpp

```diff
@@ -577,10 +578,11 @@ void SailfishUserManager::onCreatingJobFailed(SystemdManager::JobList &remaining
             // session systemd is stopped but autologin is still up and it wasn't brought down
             // TODO: What to do?
             qCWarning(lcSUM) << "Could not stop autologin, unhandled error case";
+            emit currentUserChangeFailed(m_switchUser);
```

Comment:
that "unhandled error case" logging above - are we now handling the error case i.e. should the log message be adjusted?

Reply:
Emitting that signal doesn't really do anything useful as user session is mostly stopped at this point. I don't see this signal doing any harm (if not any good either) but I would not call it handled with just this.